instrumentation/opentelemetry-instrumentation-asyncio/src/opentelemetry/instrumentation/asyncio/utils.py

```diff
@@ -0,0 +1,66 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+# pylint: disable=no-name-in-module
+from opentelemetry.instrumentation.asyncio.environment_variables import (
+    OTEL_PYTHON_ASYNCIO_COROUTINE_NAMES_TO_TRACE,
+    OTEL_PYTHON_ASYNCIO_FUTURE_TRACE_ENABLED,
+    OTEL_PYTHON_ASYNCIO_TO_THREAD_FUNCTION_NAMES_TO_TRACE,
+)
+
+
+def separate_coro_names_by_comma(coro_names: str) -> set:
+    """
+    Function to separate the coroutines to be traced by comma
+    """
+    if coro_names is None:
+        return set()
+    return set(coro_name.strip() for coro_name in coro_names.split(","))
```

Comment:
nit can use set comprehension directly I think

```suggestion
    return {coro_name.strip() for coro_name in coro_names.split(",")}
```